src/common/liboptparse/optparse.c

```diff
@@ -1216,6 +1216,8 @@ static char * optstring_append (char *optstring, struct optparse_option *o)
      *   an option with no argument (has_arg = 0), 2 characters
      *   for a required argument (has_arg = 1), and 3 chars for
      *   an optional argument "o::" (has_arg = 2)
+     *  N.B.: optional argument on short option only with
+     *   OPTPARSE_OPT_OPTIONAL_ARG_SHORTOPT
```

Comment:
s/only/also/ ?

Reply:
yeah, that is ambiguous, the comment is meant to say the optional argument will only be applied if the flag is set, I'll reword. (Also the name of the flag is wrong :facepalm: )